core/modules/book/book.install

```diff
@@ -138,6 +141,18 @@ function book_update_1002() {
   $config->save();
 }
 
+/**
+ * Creates the table to enable caching of Book navigation.
+ */
+function book_update_1003() {
```

Comment:
I get a clash with a recently merged update with the same number. This PR needs a rebase, and renaming the install hook.